examples/loadHttp/kustomization.yaml

```diff
@@ -4,12 +4,11 @@ resources:
 - https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/examples/wordpress/wordpress/deployment.yaml
 - https://github.com/knative/serving/releases/download/v0.12.0/serving.yaml # redirects to s3
 patches:
```

Comment:
removed legacy patch syntax is used.

Reply:
This is used in other examples - is it worth updating those as well?

Reply:
@cailynse  

I think I searched and fixed other examples where the removed legacy patch was used.
Could you find other places for it was used?

Reply:
Sorry! I misunderstood, I was thinking `patchesStrategicMerge` which is being used in many places